wgpu-hal/src/metal/command.rs

```diff
@@ -60,47 +74,48 @@ impl super::CommandEncoder {
                 .contains(TimestampQuerySupport::ON_BLIT_ENCODER);
 
             if !self.state.pending_timer_queries.is_empty() && !supports_sample_counters_in_buffer {
-                objc::rc::autoreleasepool(|| {
-                    let descriptor = metal::BlitPassDescriptor::new();
+                autoreleasepool(|_| unsafe {
```

Comment:
nitpick: Can we please narrow the spans that this `unsafe` block applies to? If you're aware of the rationale, a `SAFETY` comment would be great, but I know that in other places you've disclaimed knowledge of the Metal API specifically.

Reply:
Reason I did it this way is mostly the same as in https://github.com/gfx-rs/wgpu/pull/5641#discussion_r2066231610: this `unsafe` will likely disappear some day.